core-bundle/src/DataContainer/PaletteManipulator.php

```diff
@@ -45,17 +45,16 @@ public static function create(): self
      * If the legend already exists, nothing will be changed.
      *
      * @param string|array $parent
-     * @param bool         $hide
      */
-    public function addLegend(string $name, $parent, string $position = self::POSITION_AFTER, $hide = false): self
+    public function addLegend(string $name, $parent, string $position = self::POSITION_AFTER, bool $hide = false): self
```

Comment:
Wouldn't the same arguments apply here as in https://github.com/contao/contao/issues/3001 ? If this is possible after all, then we should have just changed the method signature of `FilesModel::findByPath`.

Otherwise we cannot allow this change, since it changes the method signature of a public method?

Reply:
I seriously doubt that anyone has ever passed a non-boolean value here (in contrast to #3001 where `$path` could be `null`), but I have reverted the change in 7f1606206d692382796dc9f30ee23ece18acf9fb, so that we do not have to waste time with unnecessary discussions.